Makefile

```diff
@@ -2,18 +2,16 @@
 PROJECT_NAME = cc_licenses
 STATIC_DIR = ./$(PROJECT_NAME)/static
 
-default: lint test
+default:
+	# Checks syntax and runs tests
+	pre-commit run -v -a
 
 test:
```

Comment:
Do we need both `default` and `test` if they both run the same command now? Could we just use `default` and point travis to use it?